config/functions

```diff
@@ -103,6 +103,36 @@ setup_toolchain() {
   esac
 }
 
+# unset all PKG_* vars apart from those exported by setup_toolchain, then set default values
+reset_pkg_vars() {
+  local vars var
+
+  for var in $(set | awk -F= '/^PKG_.*=/ { print $1 }'); do
```

Comment:
bash allows `for var in ${!PKG_*}; do`

Reply:
Oh... very nice, thanks! :)

Is this likely to be compatible with all versions of bash?

Reply:
I've added an extra commit using your solution which I will squash if there are no compatibility issues.